examples/guard_functions/test/tests.ts

```diff
@@ -1,149 +1,89 @@
-
-                return {
-                    Ok:
-                        counterBefore === 0 &&
-                        methodExecuted &&
-                        counterAfter === 1
-                };
-            }
-        },
-        {
-            name: 'tightlyGuarded',
-            test: async () => {
-                try {
-                    await guardFunctionsCanister.tightlyGuarded();
-                    return {
-                        Err: 'Expected tightlyGuarded function to throw'
-                    };
-                } catch (error) {
-                    return {
-                        Ok: (error as AgentError).message.includes(
-                            `Uncaught Error: Execution halted by \\"unpassable\\" guard function`
-                        )
-                    };
-                }
-            }
-        },
-        {
-            name: 'errorStringGuarded',
-            test: async () => {
-                try {
-                    await guardFunctionsCanister.errorStringGuarded();
-                    return {
-                        Err: 'Expected errorStringGuarded function to throw'
-                    };
-                } catch (error) {
-                    return {
-                        Ok: (error as AgentError).message.includes(
-                            `Uncaught Error: Execution halted by \\"throw string\\" guard function`
-                        )
-                    };
-                }
-            }
-        },
-        {
-            name: 'customErrorGuarded',
-            test: async () => {
-                try {
-                    await guardFunctionsCanister.customErrorGuarded();
-                    return {
-                        Err: 'Expected customErrorGuarded function to throw'
-                    };
-                } catch (error) {
-                    return {
-                        Ok: (error as AgentError).message.includes(
-                            `Uncaught CustomError: Execution halted by \\"throw custom error\\" guard function`
-                        )
-                    };
-                }
-            }
-        },
-        {
-            name: 'nonStringErrValueGuarded',
-            test: async () => {
-                try {
-                    await guardFunctionsCanister.nonStringErrValueGuarded();
-                    return {
-                        Err: 'Expected nonStringErrValueGuarded function to throw'
-                    };
-                } catch (error) {
-                    return {
-                        Ok: (error as AgentError).message.includes(
-                            `Uncaught Error: [object Object]`
-                        )
-                    };
-                }
-            }
-        }
-    ];
+): Test {
+    return () => {
+        it('successfully calls an unguarded method', async () => {
+            const result = await guardFunctionsCanister.identifierAnnotation();
```

Comment:
Were these from when we had a different API for guard methods? I don't exactly understand what they are doing